wrappers/desktop.go

```diff
@@ -61,6 +61,13 @@ var validDesktopFilePrefixes = []string{
 	"Keywords=",
 	"StartupNotify=",
 	"StartupWMClass=",
+	// unity extension
+	"X-Ayatana-Desktop-Shortcuts=",
+	"TargetEnvironment=",
+}
+
+var validDesktopFileSuffixes = []string{
+	"Shortcut Group]",
```

Comment:
It's a suffix per the variable name.. I think @chipaca's suggestion is enough to nail that down without a comment. If we are to comment, then we ought to say why it's like that rather than just re-stating it is like that because it is like that.